test/cubits/welcome_cubit_test.dart

```diff
@@ -0,0 +1,99 @@
+// ignore_for_file: lines_longer_than_80_chars
+
+import 'package:bloc_test/bloc_test.dart';
+import 'package:flutter_template/core/repository/project_repository.dart';
+import 'package:flutter_template/core/repository/session_repository.dart';
+import 'package:flutter_template/ui/welcome/welcome_cubit.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:get_it/get_it.dart';
+import 'package:mocktail/mocktail.dart';
+
+import '../common/mocks.dart';
+import '../common/project_helpers.dart';
```

Comment:
linter should give you an issue, fix it

Reply:
It doesn’t because this files are in the test folder that exists outside of lib. `package:flutter_template/test/common/mocks.dart` doesn’t exists

The alternative is adding this in the lib folder, but i dont think it makes much sense, since we are only using this for the tests

Reply:
What about defining rules only for the test folder?

Reply:
Do you mean use only relative paths in test files? Because you can't import the files outside of lib as `package:flutter-template/...` 
https://dart.dev/tools/pub/package-layout#public-directories 
https://stackoverflow.com/questions/64145591/how-to-import-test-util-files-inside-of-your-tests-on-dart-flutter